src/backend/distributed/planner/multi_logical_planner.c

```diff
@@ -1560,7 +1630,9 @@ ExtractRangeTableRelationWalker(Node *node, List **rangeTableRelationList)
 	foreach(rangeTableCell, rangeTableList)
 	{
 		RangeTblEntry *rangeTableEntry = (RangeTblEntry *) lfirst(rangeTableCell);
-		if (rangeTableEntry->rtekind == RTE_RELATION)
+		if (rangeTableEntry->rtekind == RTE_RELATION &&
```

Comment:
Maybe it'd be better to negatively filter views here, rather than list all the types of tables we accept?